android/engine/src/test/resources/plans/child-immunization-schedule/structure-map-register.txt

```diff
@@ -43,6 +44,25 @@ group ExtractTask(
                subject -> groupIdentifier.use = "secondary" "rule_group_identifier_use";
 
         } "rule_group_identifier";
+
+         subject -> task.partOf = create('Reference') as ref then {
+                subject -> evaluate(questionnaireResponse, $this.descendants().where(linkId='123456').answer.value.toString()) as taskId then{
```

Comment:
I don't think this will work because by the time you are generating the references you do not have any pre-generated tasks.